modules/installation-restricted-network-samples.adoc

```diff
@@ -105,14 +105,14 @@ This is required because the imagestream import process does not use the mirror
 ====
 +
 . Add any imagestreams that are not mirrored into the `skippedImagestreams` field
-of the Samples Operator configuration object. Or if you do not want to support
-any of the sample imagestreams, set the Samples Operator to `Removed` in the
-Samples Operator configuration object.
+of the Cluster Samples Operator configuration object. Or if you do not want to support
+any of the sample imagestreams, set the Cluster Samples Operator to `Removed` in the
+Cluster Samples Operator configuration object.
 +
 [NOTE]
 ====
-Any unmirrored imagestreams that are not skipped, or if the Samples Operator is
-not changed to `Removed`, will result in the Samples Operator reporting a
+Any unmirrored imagestreams that are not skipped, or if the Cluster Samples Operator is
+not changed to `Removed`, will result in the Cluster Samples Operator reporting a
 `Degraded` status two hours after the imagestream imports start failing.
```

Comment:
Pulling this comment forward from the original PR:

"Ah shoot found another item I missed. In 4.6 only, we backed off marking degraded with openshift/cluster-samples-operator@ecfb9e6

And as a replacement, it issues alerts if a) imports are failing but samples operator is periodically retrying them, b) imports are failing and samples operator does not appear to be retrying them."

@gabemontero ACK - I think I'll need to open a follow-up PR specifically for 4.6 for this one.

Reply:
Addressed for 4.6 here: https://github.com/openshift/openshift-docs/pull/26435